app/components/candidate_interface/gcse_qualification_review_component.rb

```diff
@@ -94,19 +89,38 @@ def country_row
       }
     end
 
-    def naric_row
+    def naric_statment_row
+      return nil unless FeatureFlag.active?('international_gcses') && application_qualification.qualification_type == 'non_uk'
+
+      {
+        key: 'Do you have a NARIC statment of comparability?',
```

Comment:
```suggestion
        key: 'Do you have a NARIC statement of comparability?',
```

Sorry, this was my spelling mistake!